src/index.html

```diff
@@ -6,9 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#"><img src="/src/images/logo.png" alt="moyo-logo"></a>
+      <nav class="navbar">
+        <ul class="product-list">
+          <li class="product">
+            <a class="nav-link is-active" href="#">Apple</a>
+          </li>
+          <li class="product">
+            <a class="nav-link" href="#">Samsung</a>
+          </li>
+          <li class="product">
+            <a class="nav-link" href="#">Smartphones</a>
+          </li>
+          <li class="product">
+            <a class="nav-link" href="#"
+            data-qa="hover">Laptops & Computers</a>
+
+          </li>
+          <li class="product">
+            <a class="nav-link" href="#">Gadgets</a>
+          </li>
+          <li class="product">
+            <a class="nav-link" href="#">Tablets</a>
+          </li>
+          <li class="product">
+            <a class="nav-link" href="#">Photo</a>
+          </li>
+          <li class="product">
+            <a class="nav-link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
```suggestion
      <nav class="navbar">
        <ul class="product-list">
          <li class="product">
            <a class="nav-link is-active" href="#">Apple</a>
          </li>
          
          <li class="product">
            <a class="nav-link" href="#">Samsung</a>
          </li>
          
          <li class="product">
            <a class="nav-link" href="#">Smartphones</a>
          </li>
          
          <li class="product">
            <a 
              class="nav-link" 
              href="#"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          
          <li class="product">
            <a class="nav-link" href="#">Gadgets</a>
          </li>
          
          <li class="product">
            <a class="nav-link" href="#">Tablets</a>
          </li>
          
          <li class="product">
            <a class="nav-link" href="#">Photo</a>
          </li>
          
          <li class="product">
            <a class="nav-link" href="#">Video</a>
          </li>
        </ul>
      </nav>
```

pls pay attention to your code style.
It is always a good idea to make blank lines between multiline siblings.

![image](https://github.com/mate-academy/layout_moyo-header/assets/108538921/a626307c-fa8f-4ffd-8ae3-de7b8627265a)